tests/snowpark/test_function.py

```diff
@@ -0,0 +1,249 @@
+import contextlib
+import os
+from pathlib import Path
+from tempfile import NamedTemporaryFile, TemporaryDirectory
+from textwrap import dedent
+from unittest import mock
+
+
+@mock.patch("snowflake.connector.connect")
+@mock.patch("snowcli.cli.snowpark.function.commands.snowpark_package")
+@mock.patch("snowcli.cli.snowpark.function.commands.TemporaryDirectory")
+def test_create_function(
```

Comment:
Do we want to separate tests for commands and managers?

Reply:
Yes, but I would like to do this in next step. Once I refactor procedure commands I should be able to extract common manager for functions and procedures.